src/main/java/uk/gov/hmcts/reform/sscs/utils/HearingUtils.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.reform.sscs.utils;
+
+import uk.gov.hmcts.reform.sscs.ccd.domain.ExcludeDate;
+import uk.gov.hmcts.reform.sscs.ccd.domain.HearingOptions;
+import uk.gov.hmcts.reform.sscs.ccd.domain.HearingSubtype;
+import uk.gov.hmcts.reform.sscs.model.servicehearingvalues.UnavailabilityRange;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public final class HearingUtils {
+    public static final String FACE_TO_FACE = "faceToFace";
+    public static final String TELEPHONE = "telephone";
+    public static final String VIDEO = "video";
+    public static final String PAPER = "paper";
```

Comment:
Can you put in a comment that this needs to be replaced with the reference from SSCS-10273 once released

https://github.com/hmcts/sscs-common/pull/1174/files#diff-5033a421baf6bdaa41e1d9333725554e7b7325a277d11e7aca068dc9da2e81f8R10-R13

Reply:
Done